fileio.c

```diff
@@ -0,0 +1,53 @@
+/*Mariposa Lee, reads input.txt, prints average and squares. A friend helped me through this in the morning.*/
```

Comment:
this only works for when four numbers or fewer are in the file! And then it calculates the average incorrectly